exalearn-ui/.prettierrc

```diff
@@ -0,0 +1,23 @@
+{
+    "printWidth": 120,
+    "singleQuote": true,
+    "endOfLine": "auto",
+    "trailingComma": "none",
```

Comment:
Please, add "jsxBracketSameLine": true, "useTabs": true, "tabWidth": 4, "semi": true options

Reply:
ok, i got it